web/src/stores/clips.ts

```diff
@@ -53,93 +60,105 @@ export namespace Clips {
 
   export type Action =
     | LoadAction
+    | LoadErrorAction
     | RefillCacheAction
     | RemoveClipAction
     | AchievementAction;
 
   export const actions = {
-    refillCache: () => async (
-      dispatch: Dispatch<RefillCacheAction | LoadAction>,
-      getState: () => StateTree
-    ) => {
-      const state = getState();
-      if (localeClips(state).clips.length > MIN_CACHE_SIZE) {
-        return;
-      }
+    refillCache:
+      () =>
+      async (
+        dispatch: Dispatch<RefillCacheAction | LoadAction | LoadErrorAction>,
+        getState: () => StateTree
+      ) => {
+        const state = getState();
+
+        // don't load if no contributable locale
+        if (!contributableLocales.includes(state.locale)) {
```

Comment:
We moved this logic from https://github.com/common-voice/common-voice/pull/3483/files#diff-c47fbf66be6c67df8a04429035911604d242969370b34d120f39cb668e569f7eL31